### PR TITLE
Adds the ability to skip creating new test cases if they don't already exist in testrail

### PR DESCRIPTION
**Pull request by olhado**
Added a simple flag that can be set on a per-job basis to avoid creating tests in testrail.

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/testrail/TestRailNotifier.java [hunk 1]
```diff
@@ -191,27 +196,43 @@ public Results addSuite(Testsuite suite, String parentId, ExistingTestCases exis
...
+                    if (this.createNewTestcases) {
+                        caseId = existingCases.addCase(testcase, sectionId);
+                        addResult = true;
+                    }
                 }
-
-                if (caseStatus != CaseStatus.UNTESTED){
-                    results.addResult(new Result(caseId, caseStatus, caseComment, caseTime));
+                if (addResult) {
+                    int caseStatus;
```
Comment (xirsoi) [inline src/main/java/org/jenkinsci/plugins/testrail/TestRailNotifier.java]: It looks like both code blocks were retained when you merged from upstream. You'll want to delete everything from this line to line 221.

Comment (olhado) [inline src/main/java/org/jenkinsci/plugins/testrail/TestRailNotifier.java]: Yup, I just pushed the correct upstream. Sorry about that!

**Comment by xirsoi** [comment]
You fixed the bad merge as I was typing my comment 😉

File: src/main/java/org/jenkinsci/plugins/testrail/TestRailNotifier.java (hunk 1 above)
Comment (olhado) [inline src/main/java/org/jenkinsci/plugins/testrail/TestRailNotifier.java]: Should I switch the order of the checkbox and text?

**Pull request merged**